packages/cli/src/Server.ts

```diff
@@ -1932,6 +1933,8 @@ class App {
 		// OAuth2-Credential/Auth
 		// ----------------------------------------
 
+		this.app.use(`/${this.restEndpoint}/oauth2-credential`, oauth2CredentialController);
```

Comment:
In future we can move `/oauth2-credential/auth` and `/oauth2-credential/callback` to the new controller.

Reply:
we should 😊 can you create a Tech Debt ticket for it?